lib/matplotlib/backends/backend_qt5.py

```diff
@@ -709,7 +709,7 @@ def _init_toolbar(self):
         # Esthetic adjustments - we need to set these explicitly in PyQt5
         # otherwise the layout looks different - but we don't want to set it if
         # not using HiDPI icons otherwise they look worse than before.
```

Comment:
Shouldn't this comment be removed then?

Reply:
The comment is still valuable. The code now does what the comment specifies; and it's good to know why there is the hard-coding for HiDpi. Not sure though if the hard-coding is still necessary, but as long as the following `if` statement is there, the comment should stay.

Reply:
Ah wait sorry, for some reason, I read the diff backwards. Never mind then.